src/app/(marketing)/pricing/components/ClerkPricingTable.module.css

```
@@ -0,0 +1,432 @@
+  background: color-mix(in oklab, var(--primary) 90%, black);
+  box-shadow: 0 12px 28px -14px
+    color-mix(in oklab, var(--primary) 70%, transparent);
+}
+
+.checkoutMount {
+  display: block;
+  width: 100%;
+}
+
+.cards
+  :global(.cl-pricingTableCardFooter):has(.checkoutMount > .checkoutButton)
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)